docs/static/js/manon.js

```diff
@@ -338,7 +358,7 @@
     var headers = document.querySelectorAll("header:not(.breadcrumbs)");
     for (var i = 0; i < headers.length; i++) {
       var nav = headers[i].querySelector("nav");
-      if (!(nav instanceof HTMLElement) || nav.querySelector(".menu_toggle")) {
+      if (!(nav instanceof HTMLElement) || nav.querySelector(".menu-toggle")) {
```

Comment:
Class names with a hypen are harder to access from JS land as properties because the hyphen gets interpreted as a minus sign.
The can still be accessed using bracket notation, but this might be something that throws users into a complex debugging session.

Reply:
Added the bracket notation to resolve this issue.

Reply:
And removed the brackets as discussed with @underdarknl. Since they aren't needed here.